website/src/graphql/dailp/queries.graphql

```diff
@@ -391,68 +365,14 @@ query CollectionChapter($collectionSlug: String!, $chapterSlug: String!) {
       slug
     }
     document {
-      id
-      title
-      slug
-      isReference
-      date {
-        year
-      }
-      bookmarkedOn {
-        formattedDate
-      }
-      sources {
-        name
-        link
-      }
-      audioRecording {
-        resourceUrl
-        startTime
-        endTime
-      }
-      translatedPages {
-        image {
-          url
-        }
-      }
-      chapters {
-        id
-        path
-      }
+      ...DocumentFields
     }
   }
 }
 
 query BookmarkedDocuments {
   bookmarkedDocuments {
-    id
-    title
-    slug
-    isReference
-    date {
-      year
-    }
-    bookmarkedOn {
-      formattedDate
-    }
-    sources {
-      name
-      link
-    }
-    audioRecording {
-      resourceUrl
-      startTime
-      endTime
-    }
-    translatedPages {
-      image {
-        url
-      }
-    }
-    chapters {
-      id
-      path
-    }
+    ...DocumentFields
```

Comment:
Previously, this large set of fields was requested, but only ID was used. It seems to work now by fetching the `DocumentFields` fragment and passing the data directly to the child component rendering each bookmark.